config.go

```diff
@@ -0,0 +1,37 @@
+package viewproxy
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type configRouteEntry struct {
+	Url               string   `json:"url"`
+	LayoutFragmentUrl string   `json:"layout"`
+	FragmentUrls      []string `json:"fragments"`
+}
+
+func readConfigFile(filePath string) ([]configRouteEntry, error) {
```

Comment:
I was surprised this could be used outside of the file since it's not TitleCased. But it can be used from `server.go` because they're both in the top-level `viewproxy` package, right?

Reply:
Yep, exactly!